manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -25,6 +25,11 @@ rules:
     verbs: ["update"]
     # TODO: should be on just */status but wildcard is not supported
     resources: ["*"]
+{{- end }}
+{{- if .Values.global.configValidation }}
+  - apiGroups: ["networking.istio.io"]
```

Comment:
related authority has already been there : https://github.com/istio/istio/blob/master/manifests/charts/base/templates/role.yaml#L12 

So this change is not needed.

Reply:
Ah! I see. We have a custom `ServiceAccount` that was bound only to the `ClusterRole`, missed the `RoleBinding`. Thanks for the input. Closing the PR.